wp-lazy-loading.php

```diff
@@ -134,33 +133,17 @@ function wp_lazy_loading_enabled( $tag_name, $context ) {
  * @return string Converted content with 'loading' attributes added to images.
  */
 function wp_add_lazy_load_attributes( $content, $context = null ) {
-	$tags = array();
-
-	if ( null === $context ) {
-		$context = current_filter();
-	}
-
-	if ( wp_lazy_loading_enabled( 'img', $context ) ) {
-		$tags[] = 'img';
-	}
-
-	// Experimental. Will be removed when merging unless the HTML specs are updated by that time.
-	if ( wp_lazy_loading_enabled( 'iframe', $context ) ) {
-		$tags[] = 'iframe';
-	}
-
-	if ( empty( $tags ) ) {
+	if ( ! wp_lazy_loading_enabled( 'img', $context ) ) {
 		return $content;
 	}
 
 	return preg_replace_callback(
-		'/<(' . implode( '|', $tags ) . ')(\s)[^>]+>/',
+		'/<img(\s)[^>]+>/',
```

Comment:
Don't think we need to capture the space here. Doesn't make sense to reinsert it. We only need to check for it's existence.

This is a tiny optimization, but optimization nevertheless :)

In nearly all cases capturing and reinserting the space char will make no difference at all. If the tag is formatted with line breaks between the attributes (cannot happen in post_content, comments, widget text, and pretty much all content entered by the users as it breaks wpautop), capturing and reinserting the space will "push" the attribute to the second line, but still "break" the formatting as there will be two attributes there. 

In any case, "white space" is ignored in HTML, and adding another attribute will "shuffle" the existing attributes a bit. Don't see a good reason to make the regex a tiny bit slower in order to "shuffle" the attributes in one way and not the other :)

Reply:
IMHO this should be `'/<img\s[^>]+>/'`.

Reply:
That's fine with me